pages/common/dolt-fetch.md

```diff
@@ -0,0 +1,16 @@
+# dolt fetch
+
+> Download objects and refs from another repository.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Fetch refs and update remote-tracking branches from the default remote named "origin":
+
+`dolt fetch`
+
+- Fetch refs and update remote-tracking branches from a specific remote:
```

Comment:
Perhaps this better
```suggestion
- Fetch latest changes from a specific remote upstream repository:
```

Reply:
solved in c75c5d817b331439b6fa3e264d6b0baa96923792